angular1/README.md

```diff
@@ -0,0 +1,45 @@
+# Angular Seed TypeScript
+This is based on https://github.com/angular/angular-seed.
+
+## Running
+The following are specific to TypeScript
+
+Setup TypeScript:
+```bash
+npm install typescript -g
+npm install tsd -g
+```
+Start the TypeScript compiler in watch mode (either in the `app` folder or in the `e2e-tests` folder) and **leave it running**: 
+
+```bash
+tsc --watch --p . 
+```
+
+That's it. You have typescript setup and ready to go. You can follow the steps of JavaScript ([README-JS](./README-JS.md)) from this point on in a new window starting at the install dependencies section.
+
+**TIP**: *Abbriged the remaining JS steps for a quick start:*
+```bash
+npm install 
+npm start
+```
+and visit : http://localhost:8000/app/
+
+## Upgrading
```

Comment:
is this needed for the readme? on another note "upgrading" sounded to vague to me.